blocks/api/raw-handling/shortcode-converter.js

```diff
@@ -41,7 +41,10 @@ export default function( HTML ) {
 				getBlockAttributes(
 					{
 						...blockType,
-						attributes: transform.attributes,
+						attributes: mapValues( transform.attributes, attribute => ( {
```

Comment:
Worried at all that the implementer who's fetching attributes for a block needs to be aware of this defaulting? Should this be built in `getBlockAttributes` or `createBlock` ? (Maybe again to the point of `default` case)

Reply:
What if the `undefined` case is the comment. I kind of like explicit types but I can see the concern raised here.

Reply:
Ah, so vs. the `default` case (which would also capture meta), only explicitly undefined `source`?

Reply:
Yep, exactly

Reply:
That sounds okay to me 👍